addons/html_builder/static/src/core/building_blocks/select_many2x.scss

```diff
@@ -0,0 +1,70 @@
+.o-hb-selectMany2X-wrapper {
+    // Disclaimer: the following rules essentially overrides utility classes
+    // "inherited" from the backend and apply BuilderSelect rules on top.
+    // To be improved by removing the unwanted classes and share relevant
+    // rules across the two components.
+
+    border: 0 !important; // Override backend's `border`class
+    width: 100% !important; // Override backend's `w-auto`class
+
+    div:has(> &) {
+        // Apply `o-hb-select-wrapper` classes on the unnamed parent div
+        flex: 1 0 auto;
+        min-width: 7ch; // Approx 4 characters + caret
+        max-width: 100%;
+        display: flex;
+    }
+
+    .o-hb-selectMany2X-toggle {
+        &, &.btn-light.bg-light {
```

Comment:
Meh ^^ Do we need that specific a selector?

Reply:
Unfortunately yes for now.
The alternative was to tweak the default props, but the component it's used in the backend as well.